src/lib/operators/table_scan/column_between_table_scan_impl.cpp

```diff
@@ -65,8 +65,13 @@ void ColumnBetweenTableScanImpl::_scan_generic_segment(const BaseSegment& segmen
 void ColumnBetweenTableScanImpl::_scan_dictionary_segment(const BaseDictionarySegment& segment, const ChunkID chunk_id,
                                                           PosList& matches,
                                                           const std::shared_ptr<const PosList>& position_filter) const {
-  const auto left_value_id = segment.lower_bound(_left_value);
-  const auto right_value_id = segment.upper_bound(_right_value);
+  auto left_value_id = segment.lower_bound(_left_value);
```

Comment:
The `const` is removed for consistency reasons?

Reply:
with right_value_id below, yes. No strong opinion about this one though.